packages/types/src/line-item/common.ts

```diff
@@ -0,0 +1,54 @@
+// TODO
+export type LineItemTaxLineDTO = {}
+
+// TODO
+export type LineItemAdjustmentDTO = {}
+
+export type LineItemDTO = {
```

Comment:
todo: since all the DTO are part of different domain and the domain can be replaced, maybe we can add the relations as unknown in a DTO. We can then type it when we consume it with the type we expect it to be. Maybe something to think about in general when we are referencing relation across modules.

cc @carlos-r-l-rodrigues @olivermrbl